src/components/dialogs/network-modification/composite-modification/composite-modification-edition-dialog.tsx

```diff
@@ -0,0 +1,169 @@
+    const { snackError } = useSnackMessage();
+    const selectionForCopy = useSelector((state: AppState) => state.selectionForCopy);
+    const [modifications, setModifications] = useState<NetworkModificationMetadata[]>([]);
+    const dispatch = useDispatch();
+
+    const methods = useForm<FormData>({
+        defaultValues: emptyFormData(name),
+        resolver: yupResolver(schema),
+    });
+
+    const { computeLabel } = useModificationLabelComputer();
+    const getModificationLabel = (modif: NetworkModificationMetadata) => {
+        if (!modif) {
+            return null;
+        }
+        return intl.formatMessage(
+            { id: 'network_modifications.' + modif.type },
+            {
+                ...modif,
+                ...computeLabel(modif),
+            }
+        );
+    };
+
+    const renderNetworkModificationsList = () => {
+        return (
+            <>
+                {modifications && (
+                    <List sx={unscrollableDialogStyles.scrollableContent}>
+                        {modifications.map((modification: NetworkModificationMetadata) => (
+                            <>
+                                <ListItem key={modification.uuid}>
+                                    <Box>{getModificationLabel(modification)}</Box>
+                                </ListItem>
+                                <Divider component="li" />
+                            </>
+                        ))}
+                    </List>
+                )}
+            </>
+        );
+    };
+
+    useEffect(() => {
+        setIsFetching(true);
+        getCompositeModificationContent(compositeModificationId)
+            .then((response) => {
+                if (response) {
+                    setModifications(response);
+                }
+            })
+            .catch((error) => {
+                snackError({
+                    messageTxt: error.message,
+                    headerId: 'cannotRetrieveCompositeModification',
+                });
+            })
+            .finally(() => setIsFetching(false));
+    }, [compositeModificationId, name, snackError]);
+
+    const closeAndClear = (event?: SyntheticEvent) => {
+        onClose(event);
+    };
+
+    const onSubmit = (formData: FormData) => {
+        saveCompositeModification(compositeModificationId, formData[FieldConstants.NAME])
+            .then(() => {
+                if (selectionForCopy.sourceItemUuid === compositeModificationId) {
+                    dispatch(setSelectionForCopy(NO_SELECTION_FOR_COPY));
+                    broadcastChannel.postMessage({
+                        NO_SELECTION_FOR_COPY,
+                    });
+                }
+                closeAndClear();
+            })
+            .catch((errorMessage) => {
+                snackError({
+                    messageTxt: errorMessage,
+                    headerId: 'contingencyListEditingError',
```

Comment:
contingencyListEditingError ?
it is copy paste forgetten

Reply:
Oops ! Well seen thank you. Done.